### ISSUE TITLE
Warehouse report - submit and post submit fixes  

### ISSUE THREAD

**Issue by srisub** [ShoppinPal/warehouse#132]
Currently the submit stays disabled since the logic is checking that all items haven't been processed. Often the warehouse won't be able to fulfill all items and so they will just do the ones they can and then submit the report at that point. Always keep the button enabled and allow to submit even if all rows haven't been processed. 

Just add a pop with a message "Would you like to submit the order or work on a few more items?" and then have 2 buttons "Submit" and "Cancel". Cancel should just keep them on the report screen and submit should take the user to the wareh landing with the report now showing in 'To Be Received' state. Note that this is a separate bug right now where the report doesn't move to Receive status (appropirate status color change doesn't show up on landing) once warehouse user submits.

**Comment by ayushranjan-planetria** [comment]
Hi @srisub , I believe this issue is similar to Issue #128 , I have fixed #128  and created PR, thanks.
cc @pulkitsinghal

**Comment by srisub** [comment]
Good catch @ayushranjan-planetria, I updated it to only post submit fixes - assuming you didn't tackle these 2 yet
